tests/regression_tests/mgxs_library_hdf5/test.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, *args, **kwargs):
         self.mgxs_lib.by_nuclide = False
 
         # Test all MGXS types
-        self.mgxs_lib.mgxs_types = openmc.mgxs.MGXS_TYPES + \
+        self.mgxs_lib.mgxs_types = openmc.mgxs.MGXS_TYPES[:-1] + \
```

Comment:
Would leave currents in here too.

Reply:
The problem here is that the domain type imposed is 'material', but current can only work on a 'mesh' domain.

Reply:
If adapting the current unit test is not possible, I think you need to create another.
btw, _hdf5_key does not seem to be set for "current" (also "total" and "transport"). Can you please check what it defaults to in a hdf5 file?

Reply:
It defaults to "current". Do I need to add that information anywhere? 

I think I will expand mgxs_library_mesh to include all the MGXS function tests. It's the only test that handles the mesh domain, so this way all the classes can be tested for how they handle a mesh domain in every function. This seems to be missing anyway.

Reply:
I just extended mgxs_library_condense and mgxs_library_hdf5 to include current by modifying their domain from 'material' to 'mesh'. So now both those functionalities are tested for current.